acq4/devices/AxoPatch200/devGuiTemplate.py

```diff
@@ -24,7 +24,8 @@ def _translate(context, text, disambig):
         return QtGui.QApplication.translate(context, text, disambig)
 
 class Ui_devGui(object):
-    def setupUi(self, devGui):
```

Comment:
This file is auto-generated from `devGuiTemplate.ui`  (as the header indicates, changes to this file will be overwritten the next time it is regenerated). Instead, I would move the `modeCombo.setItemText` calls to `AxoPatch200.py`, and use Qt Designer to remove the combo box items from the `.ui` file. Finally, run `pyuic4 devGuiTemplate.ui > devGuiTemplate.py` to regenerate the python file.